app/controllers/searches_controller.rb

```diff
@@ -5,18 +5,55 @@ class SearchesController < CatalogController
 
   def index
     @search = FileSearch.new(params, catalog_query: self)
-    @results = @search.results
+    get_audios
```

Comment:
Can we use a better prefix here?  `get` is really overloaded in a lot of places.  Maybe `fetch` or `find`?